src/routes/AddedContactRoutes.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import AddedContactsController from '../controllers/AddedContactsController';
+
+const router = express.Router();
+router.get('/addedContacts', function (request, response, next) {
+  AddedContactsController.list(request, response, next);
+});
```

Comment:
You don't have to create another function here since the controller function has the same signature. You can pass the function of the controller directly: 
```
router.get('/addedContacts', AddedContactsController.list);
```